src/utils.ts

```diff
@@ -1,7 +1,42 @@
-import _ from "lodash"
-import { isObject, isString } from "lodash"
 import { LogflareUserOptionsI } from "logflare-transport-core"
 
+function isObject(value?: unknown): value is object {
+  return typeof value === "object" && value !== null
+}
+
+function isString(value?: unknown): value is string {
+  return typeof value === "string"
+}
+
+/**
+ * Takes an object and removes all falsy properties
+ */
+function removeFalsy<TInput>(input: TInput): TInput {
```

Comment:
I'm matching the old functionality of `_.pickBy({ ... }, (x) => x)` with this function, [which is to remove all _falsy_ values](https://lodash.com/docs/4.17.11#pickBy) (including things like 0 and ""). I can update this to be `removeNullAndUndefined` if you think that would be more desirable, but I just didn't want to introduce changes that might have downstream affects.